h3i/Cargo.toml

```diff
@@ -12,7 +12,7 @@ license = { workspace = true }
 keywords = { workspace = true }
 categories = { workspace = true }
 readme = "README.md"
-rust-version = "1.83"
+rust-version = "1.85"
```

Comment:
TBH I think we can just remove `rust-version` from h3i... it doesn't seem like it provides much value.